test_config.yaml

```diff
@@ -121,17 +103,24 @@
             id: syn13363443
             format: csv
         final_format: json
-        custom_transformations: 1
+        custom_transformations:
+          adjusted_p_value_threshold: 0.05
+          protein_level_threshold: 0.05
+        column_rename:
+          ensg: ensembl_gene_id
         provenance:
-          - syn26868788
+          - syn25953363
+          - syn12514826
+          - syn12514912
+          - syn18689335
+          - syn35221005
+          - syn27211942
           - syn12540368
           - syn27211878
           - syn13363443
         agora_rename:
-          eqtl: haseqtl
-          igap: isIGAP
+          symbol: hgnc_symbol
```

Comment:
I think we still need these agora_renames too:
* eqtl: haseqtl
* igap: isIGAP

Reply:
Background: when creating `gene_metadata` and `gene_info` were separate processes, the `gene_metadata` trasnform created the column `igap` and re-named the column `haseqtl` to `eqtl` (hard-coded in the transform), and both were then re-named with `agora_rename` in `gene_info`. I had removed this when combining the processes and just created `isIGAP` and did not rename `haseqtl`, which comes directly from the eqtl data set. 

Per [this JIRA task](https://sagebionetworks.jira.com/browse/AG-300), I've changed the name of `isIGAP` to `is_igap`, and then added an `agora_rename` for it. I used `column_rename` to change the eqtl column `haseqtl` to `has_eqtl`. Then it is renamed with `agora_rename`. Eventually both `agora_renames` can be removed when Agora is ready for the snake case variable names, while the transform shouldn't need to change. 

Commit [5c5dc71e9f5b92ab8b226faaf2877e8a35360b17](https://github.com/jaclynbeck-sage/agora-data-tools/commit/5c5dc71e9f5b92ab8b226faaf2877e8a35360b17).